packages/network/connman/package.mk

```diff
@@ -44,6 +44,15 @@ PKG_CONFIGURE_OPTS_TARGET="--srcdir=.. \
                            --with-systemdunitdir=/usr/lib/systemd/system \
                            --disable-silent-rules"
 
+case "$WIREGUARD_SUPPORT" in
+  yes)
+    PKG_CONFIGURE_OPTS_TARGET+=" --enable-wireguard=builtin"
+    ;;
+  *)
+    PKG_CONGIGURE_OPTS_TARGET+=" --disable-wireguard"
+    ;;
+esac
```

Comment:
Using a case for a `yes`/not `yes` test is perhaps a little excessive - normally we handle this with:
```
if [ "$WIREGUARD_SUPPORT" = "yes" ]; then
  PKG_CONFIGURE_OPTS_TARGET+=" --enable-wireguard=builtin"
else
  PKG_CONGIGURE_OPTS_TARGET+=" --disable-wireguard"
fi
```
I realise there's a `case` statement below, maybe one could argue it's a slightly more complicated test which justifies the use of `case`, but we don't need to use `case` for `yes`/not `yes` (and shouldn't, for reasons of consistency).

Reply:
I was using an if block originally but changed it to case to match the one below. No issue to change back .. done.